mage_ai/frontend/components/ServerTimeDropdown/index.style.tsx

```diff
@@ -1,12 +1,25 @@
 import styled from 'styled-components';
 
+import Button from '@oracle/elements/Button';
 import dark from '@oracle/styles/themes/dark';
 import { BORDER_RADIUS } from '@oracle/styles/units/borders';
 import { UNIT } from '@oracle/styles/units/spacing';
 
 const SHARED_CELL_PADDING = '12px 20px';
 const SHARED_GAP = '1px';
 
+export const ButtonStyle = styled(Button)<{ active?: boolean }>`
+  background-color: transparent;
+
+  ${props => (props.active) && `
+    background-color: ${(props.theme.background || dark.background).dashboard};
+  `};
+
+  &:hover {
```

Comment:
Can we add a `highlightOnHoverAlt` prop to the `Button` element so we can potentially use it for other places? We already have a `highlightOnHover` prop, but it highlights buttons with a very light background, so this would be another option for highlighting a button dark.